providers/softlayer/src/main/java/org/jclouds/softlayer/domain/ContainerVirtualGuestConfiguration.java

```diff
@@ -37,6 +42,9 @@
 
    public static final String SWAP_DEVICE = "1";
 
+   @Resource
+   Logger logger = Logger.NULL;
```

Comment:
This is not (or it shouldn't be) a Guice managed object, so this will always be `Logger.NULL`, which does nothing. You can remove it.